src/data/posts/fountain-beta.md

```diff
@@ -0,0 +1,88 @@
+Today, we're proud to announce that we open the beta version of the Yeoman Fountain generators.
+
+Basically, it's a new, state of the art, Yeoman generator. It's also, to our knowledge, the most ambitious one as it's not a generator for one technology stack but perhaps one to rules them all. More seriously, one for any webapp with the most popular frameworks and tools.
+
+Using this generator will ask you for several, very structurant, questions: the Web framework you want to use, the module management, the JS and CSS preprocessors you want.
+
+Our goal is for you to be able from a single entry point to start any Web project. That you could have the confidence that whatever the technologies you want to start with, you can count on the best, clean and efficient starter project.
+
+## Main focuses
+
+### Options
+Fountain is a new Yeoman generator allowing the user to choose the most structurant technologies you'll use:
+- Web framework
+
+  That's a key point of Fountain, you can choose either Angular 1, 2 or React and we'll provide us a starter app with this framework. And we mean a really adapted project with dev tools and test environment for the framework you chose.
+
+- Module and dependency management
+
+  This is certainly the second most structurant choice you have to make starting a Webapp these days. Webpack, SystemJS, any of them ? And believe us, this choice pull a lot of tooling and configurations to make. Fountain provide this simple choice and configure your project following it.
+
+- JavaScript version
+
+  It's became rare to start a webapp in pure browser compatible JavaScript. Everyone is about Babel or TypeScript but this needs tooling. Here you just have to choose.
+
+- Style (CSS) meta-language
+
+  The oldest preprocessors area. You just have to choose Sass or Less (or nothing).
+
+### Developer eXperience
+
+The tooling proposed is focused on offering the best developer experience in order that modern Web development benefits from the extremly dynamic ecosystem instead of creating fatigue.
```

Comment:
The proposed tooling is focused on offering the best dev experience. Now the modern web development take benefit from the extremely dynamic ecosystem instead of creating fatigue.